lib/git_reflow/git_helpers.rb

```diff
@@ -27,26 +27,41 @@ def get_first_commit_message
       run('git log --pretty=format:"%s" --no-merges -n 1', loud: false).strip
     end
 
-    def push_current_branch
-      run_command_with_label "git push origin #{current_branch}"
+    def push_current_branch(remote = 'origin')
```

Comment:
Sure. Great suggestions! It is a good idea to pass in the options hash when it is unclear what the parameters of the method should be.